src/main/java/frc/robot/Constants.java

```diff
@@ -29,7 +29,8 @@ public static final class kGripper {
     public static final int intakeVelCone = 4;
     public static final int intakeVelKube = 3;
     public static final int ejectVel = -3;
-    public static final int holdingVolage = 5;
+    public static final int holdingVolageCone = 3;
+    public static final int holdingVolageKube = 4;
```

Comment:
Nit. In VSCode, place the cursor on the variable name and use F2 to rename it everywhere.
```suggestion
    public static final int holdingVoltageCone = 3;
    public static final int holdingVoltageKube = 4;
```